src/config/dynamic-theme-fixes.config

```diff
@@ -11641,6 +11641,9 @@ div[class*="bym"][role="navigation"] {
 ::-webkit-scrollbar {
     background-color: transparent !important
 }
+.aRg, .aRj {
+  color: var(--darkreader-neutral-background);
```

Comment:
```suggestion
.aRg,
.aRj {
  color: ${white} !important;
```

Reply:
Hmm but it is white before, we don't want it to be white, then it becomes unreadable 🤔

Reply:
`${...}` is Dark Reader specific CSS syntax. Give it a try 😉